code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -9,11 +9,14 @@
 				new /obj/item/clothing/mask/gas/voice(src)
 				new /obj/item/weapon/card/id/syndicate(src)
 				new /obj/item/clothing/shoes/syndigaloshes(src)
+				new /obj/item/weapon/storage/box/syndie_kit/surveillance(src)
+				new /obj/item/weapon/dnascrambler(src)
 
 			if("stealth")
 				new /obj/item/weapon/gun/energy/crossbow(src)
 				new /obj/item/weapon/pen/paralysis(src)
 				new /obj/item/device/chameleon(src)
+				new /obj/item/weapon/soap/syndie(src)
```

Comment:
As I see it, it's because a few of the kits might not equal the number of telecrystals the bundle is worth- though I have no idea why the soap would be part of the stealth bundle.